apps/qubit/modules/settings/actions/adAction.class.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class SettingsadAction extends DefaultEditAction
```

Comment:
SettingsActiveDirectoryAction might be more explicit.

Reply:
True, but then for consistency it might be best to change all the other instances of ad to activedirectory... It could be done. Do you think it would get confused with anything else?

Reply:
Yeah, other instances would have to be changed too. SettingsadAction isn't camel-case. SettingsADAction could work too.

Reply:
Hehe... I have learned a new phrase today... I have CamelCaseified the class names...